x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/SqlQueryRequest.java

```diff
@@ -72,7 +72,7 @@ public SqlQueryRequest(String query, List<SqlTypedParamValue> params, QueryBuild
 
     @Override
     public ActionRequestValidationException validate() {
-        ActionRequestValidationException validationException = null;
+        ActionRequestValidationException validationException = super.validate();
```

Comment:
Why wasn't this called here before? The code we had was a bug?

Reply:
The overridden function (in `action/AbstractSqlQueryRequest.java`) wasn't implemented before (and now it checks client's version).